beets/dbcore/db.py

```diff
@@ -594,20 +604,31 @@ def __iter__(self):
             # Objects are pre-sorted (i.e., by the database).
             return self._get_objects()
 
-    def _make_model(self, row):
-        # Get the flexible attributes for the object.
+    def _get_flex_attrs(self, ids):
+        # Get the flexible attributes for all ids.
```

Comment:
Can we get a detailed docstring for this new method?

Reply:
I'll extend it for the next iteration of the patch :)